app/views/pages/moving-guide/civilians.html.erb

```diff
@@ -1,11 +1,331 @@
+
+      <%= render layout: 'shared/pro_tip' do %>
+        <strong>Pro-Tip:</strong> If you're traveling with pets, you can check out pet-friendly hotels on the Web at <%= link_to 'www.petswelcome.com', 'https://www.petswelcome.com/' %>.
+      <% end %>
+
+      <p>If you’re moving <%= abbr_tag('oconus') %>, make sure everyone in your family has a valid passport unless you’re headed to Hawaii or Alaska. Family members not listed on the Orders may need visas. When you arrive, expect a few in-processing steps. Depending on your host country, this could include briefings at different offices, and trips to different bases, immediately upon arrival. <%= link_to 'Transportation Office Locator', offices_path %>
+
+      <p>During your travel keep any receipts of your out of pocket expenses,  things like hotel receipts, rental car, gasoline, tolls, etc. that may be needed when you file your voucher at the end. Visit the <%= link_to 'Reimbursements & Vouchers', page_path('moving-guide/reimbursements') %> page for details.</p>
+
+      <%= render layout: 'shared/pro_tip' do %>
+        <strong>Pro-Tip:</strong> Set up a system to organize your receipts during your move, like an accordion file with a folder for each type.
+      <% end %>
+
+      <p>When moving, most, but not all of your entitlements, are subject to taxation. The <%= link_to 'Relocation Income Tax Allowance (RITA)', 'https://www.dfas.mil/civilianemployees/civrelo/relocationincometax.html' %> will reimburse you for most of those taxes. But you will not be reimbursed for <%= abbr_tag('fica') %> tax. Your Transportation Office can help navigate those reimbursements.</p>
+
+      <p>
+        <strong><em>How will your stuff get there?</em></strong><br>
+        Keep the lines of communication open and available for the driver, for <%= abbr_tag('conus') %> moves. If your driver has any updates, they may try to contact you as they make their way to your next destination and it’s important you are available to answer any questions that they may have.
+      </p>
+
+      <p>If you’re doing a door-to-door, it’s important that you arrive before the driver. If the driver arrives at your new residence before you do, they may have to put your items in storage, depending on their schedule. For those whose items are going into storage, you have more flexibility with your timing.</p>
+    </div>
+
+    <div id="pets" class="article-section">
+      <h4>Traveling with Pets</h4>
+
+      <p>First, research your new destination’s rules and regulations about bringing pets. Start by contacting your <%= link_to 'local transportation office', offices_path %> to see if there are any breed restrictions, quarantine requirements, and timeline for vaccinations. Check the <%= link_to 'Defense Travel’s Pet site', 'http://www.defensetravel.dod.mil/site/faqpet.cfm' %> for more information.</p>
```

Comment:
"Check the Defense Travel’s Pet site for more information" sounds awkward to my ears. Perhaps "Check the DoD Center for Travel Excellence's Pets FAQ for more information?"

Reply:
how about Defense Travel Management Office’s Pet FAQ